winch/filetests/src/disasm.rs

```diff
@@ -52,6 +52,12 @@ fn disassembler_for(isa: &dyn TargetIsa) -> Result<Capstone> {
             .build()
             .map_err(|e| anyhow::format_err!("{}", e))?,
 
+        Architecture::Aarch64 { .. } => Capstone::new()
+            .arm64()
+            .mode(arch::arm64::ArchMode::Arm)
+            .build()
+            .map_err(|e| anyhow::format_err!("{}", e))?,
```

Comment:
You may want to call `.set_skipdata(true)` on the built disassembler if you want to use inline constant pools. Otherwise capstone will stop disassembling at the first instruction it can't disassemble (eg due to being a constant pool)